docs/brightsky.yml

```diff
@@ -664,19 +669,25 @@ components:
       in: query
       description: DWD station ID, typically five alphanumeric characters, e.g. `P0036`. You can supply multiple station IDs separated by commas, ordered from highest to lowest priority.
       schema:
-        type: string
+        type: array
+        items:
+          type: string
```

Comment:
Can you add `explode: false` here (on the same level as `in: query`) and similarly in the below parameters?

This is not strictly necessary as our web interface supports the exploded format (the default serialization as per [this page of the OpenAPI docs](https://swagger.io/docs/specification/serialization/#query) but not as per [this page of the OpenAPI docs](https://swagger.io/docs/specification/describing-parameters/#schema-vs-content) :sweat_smile:), but the non-exploded format matches the `description`.